open-x-htb.js

```diff
@@ -104,65 +105,79 @@ function OpenXHtb(configs) {
      * @return {object}
      */
     function __generateRequestObj(returnParcels) {
-      var callbackId = '_' + System.generateUniqueId();
+        var callbackId = '_' + System.generateUniqueId();
 
-      var auidString = '';
-      var ausString = '';
+        var auidString = '';
+        var ausString = '';
 
-      /**
+        /**
        * @typedef {{
        *  applies: boolean,
        *  consentString: string
        *  }}
        */
-      var gdprConsent = ComplianceService.gdpr.getConsent();
-      var gdprPrivacyEnabled = ComplianceService.isPrivacyEnabled();
-
-
-
-      for (var i = 0; i < returnParcels.length; i++) {
-        auidString += returnParcels[i].xSlotRef.adUnitId.toString() + ',';
-        ausString += Size.arrayToString(returnParcels[i].xSlotRef.sizes, ',') + '|';
-      }
-
-      auidString = auidString.slice(0, -1);
-      ausString = ausString.slice(0, -1);
-
-      var queryObj = {
-        auid: auidString,
-        aus: ausString,
-        ju: Browser.getPageUrl(),
-        jr: Browser.getReferrer(),
-        ch: configs.charset,
-        tz: System.getTimezoneOffset(),
-        bc: 'hb_ix',
-        be: 1,
-        res: Size.arrayToString([
-          [Browser.getScreenWidth(), Browser.getScreenHeight()]
-        ]),
-        tws: Size.arrayToString([
-          [Browser.getViewportWidth(), Browser.getViewportHeight()]
-        ]),
-        ifr: Browser.isTopFrame() ? 0 : 1,
-        callback: 'window.' + SpaceCamp.NAMESPACE + '.OpenXHtb.adResponseCallbacks.' + callbackId,
-        cache: new Date().getTime()
-      };
-
-      if (gdprPrivacyEnabled) {
-        if (gdprConsent.consentString !== void(0)) {
-          queryObj.gdpr_consent = gdprConsent.consentString;
+        var gdprConsent = ComplianceService.gdpr.getConsent();
+        var gdprPrivacyEnabled = ComplianceService.isPrivacyEnabled();
+
+        var tradeDeskId;
```

Comment:
Set `tradeDeskId` to `null` on declaration,  or this variable will retain the value from the previous iteration of the outer loop.